modules/component/src/core/cd-aware/nameToStrategy.ts

```diff
@@ -0,0 +1,22 @@
+import { Observable } from 'rxjs';
+import { distinctUntilChanged, map, startWith } from 'rxjs/operators';
+import {
+  CdStrategy,
+  DEFAULT_STRATEGY_NAME,
+  StrategySelection,
+} from './strategy';
+
+export function nameToStrategy<U>(strategies: StrategySelection<U>) {
+  return (o$: Observable<string>): Observable<CdStrategy<U>> => {
+    return o$.pipe(
+      distinctUntilChanged(),
+      startWith(DEFAULT_STRATEGY_NAME),
```

Comment:
After going through Ben's comments, should the `startWith` go before the `distinct`?

Reply:
Fixed!